src/serde/thunk.rs

```diff
@@ -1,13 +1,7 @@
 use base64::{DecodeError, URL_SAFE};
 use percent_encoding::{percent_decode_str, utf8_percent_encode, AsciiSet, CONTROLS};
-use serde::{ser::Error as _, Deserialize, Deserializer, Serialize, Serializer};
-use std::{
-    borrow::Cow,
-    fmt::{Display, Formatter},
-    num::ParseIntError,
-    str::Utf8Error,
-    string::FromUtf8Error,
-};
+use serde::{__private::Formatter, ser::Error as _, Deserialize, Deserializer, Serialize, Serializer};
```

Comment:
wrong import

Reply:
__private::Formatter is needed, this was an issue from some time ago but this import randomly broke and suddenly needed the __private module before it for some reason, I can test if it is still needed.